_layouts/production.html

```diff
@@ -1,35 +1,89 @@
+                  {% else %}
+                  <div class="item" data-slide-number="{{forloop.index }}">
+                    {% endif %}
+                    <img src="{{site.url}}/{{image.url}}"></div>
+                    {% endfor %}
+                  </div><!-- Carousel nav -->
+                  <a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
+                    <span class="glyphicon glyphicon-chevron-left"></span>
+                  </a>
+                  <a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
+                    <span class="glyphicon glyphicon-chevron-right"></span>
+                  </a>
+                </div>
+              </div>
+              
+              <div class="col-sm-4" id="carousel-text"></div>
+              
+              <div class="show-info">
+                <h1>{{ page.title }}</h1>
+                <b><span class="date">
+                  {{ page.dates }} | {{page.location}}
+                </span></b>
+                <h4 class="page-synopsis">{{page.synopsis}}</h4>
+              </div>
+            </div>
+          </div>
+        </div><!--/Slider-->
+        
+        <div class="row hidden-xs" id="slider-thumbs">
+          <!-- Bottom switcher of slider -->
+          <ul class="hide-bullets">
+            {% for image in page.images %}
+            <li class="col-sm-2">
+              <a class="thumbnail" id="carousel-selector-{{forloop.index0}}"><img src="{{site.url}}/{{image.url}}"></a>
+            </li>
+            {% endfor %}
+          </ul>
+        </div>
+      </div>
+    </div>
+    
+    
+    <div class="row">
+      <div class="col-sm-12 col-md-6">
+        <h2>Production Team:</h2>
+        <ul class="hide-bullets">
+          {% for production in page.production %}
           {% if production.bio_url %}
-            <li>{{production.title}}: <a href="{{production.bio_url}}">{{production.name}}</a></li>
+          <li class="actor-name">{{production.title}}: <a href="{{production.bio_url}}">{{production.name}}</a></li>
           {% else %}
-            <li>{{production.title}}: {{production.name}}</li>
+          <li class="actor-name">{{production.title}}: {{production.name}}</li>
           {% endif %}
-        {% endfor %}
-        </ul>
-        <h3>Cast:</h3>
-        <ul>
-        {% for cast in page.cast %}
-          {% if cast.actor_bio_url %}
-            <li><a href="{{cast.actor_bio_url}}">{{cast.actor}}</a> as {{cast.role}}</li>
-          {% else %}
-            <li>{{cast.actor}} as {{cast.role}}</li>
-          {% endif %}
-        {% endfor %}
+          {% endfor %}
         </ul>
       </div>
-  </div>
-
-{{content}}
+      <div class="col-sm-12 col-md-6">
+        <h2>Cast:</h4>
+          <ul class="hide-bullets">
+            {% for cast in page.cast %}
+            {% if cast.actor_bio_url %}
```

Comment:
You might nest this if statement one level up, since it's nested in the for loop.